website/content/docs/connect/transparent-proxy.mdx

```diff
@@ -118,7 +118,7 @@ spec:
 
 In other environments, transparent proxy can be enabled via Proxy Defaults and Service Defaults config entries, or via
 the proxy service registration:
-```
+```json
```

Comment:
Our syntax highlighting library supports `hcl` as a language. These code blocks could be changed to that so that they're correctly highlighted.